src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveRptTransactionsControllerTest.java

```diff
@@ -16,10 +16,12 @@
 import uk.gov.companieshouse.api.ApiClient;
```

Comment:
Is there a test in this ControllerTest file which checks for the MULTI_YEAR_FILER being set to false and asserting the model attribute is also false?

Reply:
Discussed this with Aaron and as we are mocking that functionality about setting a multi year filer, it would not be useful to assert whether the value is false or not